src/main/java/com/cedarsoftware/util/io/JsonReader.java

```diff
@@ -953,6 +1034,26 @@ protected Object convertParsedMapsToJava(JsonObject root)
         }
     }
 
+    private static String safeTrimToNull(String string) {
+        if (string == null) {
+            return null;
+        }
+
+        String trimmedString = string.trim();
+        return trimmedString.isEmpty() ? null : trimmedString;
+    }
+
+    public Object newInstance(Class<?> c, JsonObject jsonObject) {
```

Comment:
I plan to delete newInstance() from JsonReader.  Internally, all the code should be using either createInstance() or MetaUtils.newInstance().  Let me know if you have any heart burn with me removing this.